feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -25,12 +48,62 @@ export class CustomerEmulationComponent implements OnInit, OnDestroy {
     );
     this.isCustomerEmulationSessionInProgress$ =
       this.asmComponentService.isCustomerEmulationSessionInProgress();
+
+    this.subscription.add(
+      this.cartId.valueChanges.subscribe((response: string) => {
+        this.cartIdExists = response.trim().length > 0;
+      })
+    );
+
+    this.subscription.add(
+      this.activeCartFacade.getActiveCartId().subscribe((response) => {
+        if (response) {
+          this.cartId.setValue(response);
+        }
+      })
+    );
```

Comment:
100% sure to yield an error because the active cart, by definition, is already bound to the current customer.

Reply:
The use case is that the Product Owner would like to display the active cart in the text box so that the Agent knows there is currently a cart assigned to them. I just added functionality that if the Cart id entered or loaded is the previous assigned cart, then the Assign cart to customer button is disabled to prevent an Agent from trying to assign to a customer

Reply:
The active cart id is already displayed on the cart page should the agent want to see it.
Moreover, it is confusing for the agent to see a number in the cart binding input field that:
- he don't know what this cart number is or means
- doesn't understand why he can't submit what appears to be a default value/suggestion.

Reply:
After some discussions: the feature is meant to replicate the ACC behaviour, so it stays.